google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -70,11 +74,36 @@
 
   private ArtifactPointer artifactPointer;
   private CommonModel commonModel;
+  private Sdk devAppServerJdk;
 
   private AppEngineModelSettings settings = new AppEngineModelSettings();
 
+  public AppEngineServerModel() {
+    initDefaultJdk();
+  }
+
+  @Nullable
+  private Sdk getCurrentProjectJdk() {
+    Project project = ProjectUtil.guessCurrentProject(null);
+    ProjectSdksModel projectJdksModel = new ProjectSdksModel();
+    projectJdksModel.reset(project);
```

Comment:
This seems to be the intended way to use this class. `reset` just initializes the ProjectSdksModel with respect to the project. Other uses of this class/method:
* https://github.com/JetBrains/intellij-community/blob/master/java/idea-ui/src/com/intellij/ide/util/projectWizard/SdkSettingsStep.java#L82
* https://github.com/JetBrains/intellij-community/blob/master/java/idea-ui/src/com/intellij/ide/util/projectWizard/JdkChooserPanel.java#L167